catalog-items.html

```diff
@@ -0,0 +1,371 @@
+                            </figure>
+                        </a>
+                    </article>
+                </li>
+                
+                <li class="products-item">
+                    <article class="product">
+                        <a href="catalog-items.html" class="product-link">
+                            <figure>
+                                <img src=".//img/item3.svg" alt="picture description" width="200" height="200" loading="lazy" class="product-img">
+                                <figurecaption class="item-name">SKISIRS#02</figurecaption>
+                            </figure>
+                        </a>
+                    </article>
+                </li>
+
+                <li class="products-item">
+                    <article class="product">
+                        <a href="catalog-items.html" class="product-link">
+                            <figure>
+                                <img src=".//img/item4.svg" alt="picture description" width="200" height="200" loading="lazy" class="product-img">
+                                <figurecaption class="item-name">SKISIRS#02</figurecaption>
+                            </figure>
+                        </a>
+                    </article>
+                </li>
+
+                <li class="products-item">
+                    <article class="product">
+                        <a href="catalog-items.html" class="product-link">
+                            <figure>
+                                <img src=".//img/item5.svg" alt="picture description" width="200" height="200" loading="lazy" class="product-img">
+                                <figurecaption class="item-name">SKISIRS#02</figurecaption>
+                            </figure>
+                        </a>
+                    </article>
+                </li>
+
+                <li class="products-item">
+                    <article class="product">
+                        <a href="catalog-items.html" class="product-link">
+                            <figure>
+                                <img src=".//img/item6.svg" alt="picture description" width="200" height="200" loading="lazy" class="product-img">
+                                <figurecaption class="item-name">SKISIRS#02</figurecaption>
+                            </figure>
+                        </a>
+                    </article>
+                </li>
+            </ul>      
+
+            <ul class="pagination hidden"> 
+                <li class="pagination-item">
+                    <a href="#" class="pagination-item">&laquo;</a>
+                </li>
+
+                <li class="pagination-item">                    
+                    <a href="#" class="pagination-item">&raquo;</a>
+                </li>
+            </ul>
+        </section>
+    </main>
+
+    <aside class="joinus">
+        <header class="joinus-heading">
```

Comment:
header it not necessary for all headings 

> The <header> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents introductory content, typically a group of introductory or navigational aids. It may contain some heading elements but also a logo, a search form, an author name, and other elements.